eureka-client-archaius2/src/main/java/com/netflix/discovery/guice/LocalDevEurekaClientModule.java

```diff
@@ -1,29 +1,29 @@
 package com.netflix.discovery.guice;
 
 import com.google.inject.AbstractModule;
-import com.google.inject.Scopes;
 import com.netflix.appinfo.AbstractInstanceConfig;
 import com.netflix.appinfo.ApplicationInfoManager;
+import com.netflix.appinfo.EurekaArchaius2InstanceConfig;
 import com.netflix.appinfo.EurekaInstanceConfig;
 import com.netflix.appinfo.InstanceInfo;
+import com.netflix.appinfo.providers.Archaius2VipAddressResolver;
 import com.netflix.appinfo.providers.EurekaConfigBasedInstanceInfoProvider;
+import com.netflix.appinfo.providers.VipAddressResolver;
 import com.netflix.archaius.api.Config;
 import com.netflix.discovery.DiscoveryClient;
 import com.netflix.discovery.EurekaArchaius2ClientConfig;
-import com.netflix.discovery.EurekaArchaius2InstanceConfig;
 import com.netflix.discovery.EurekaClient;
 import com.netflix.discovery.EurekaClientConfig;
 import com.netflix.discovery.shared.transport.EurekaArchaius2TransportConfig;
 import com.netflix.discovery.shared.transport.EurekaTransportConfig;
-import com.netflix.eventbus.impl.EventBusImpl;
-import com.netflix.eventbus.spi.EventBus;
 
 /**
  * @author David Liu
  */
-public class LocalDevEurekaClientModule extends AbstractModule {
+public final class LocalDevEurekaClientModule extends AbstractModule {
```

Comment:
Don't see a lot of differences between this and Ec2EurekaClientModule. Should one install the other, or have some kind of inheritance to avoid the duplicate code?